dotcom-rendering/src/web/browser/sentryLoader/init.ts

```diff
@@ -3,14 +3,20 @@ import { isAdBlockInUse } from '@guardian/commercial-core';
 import { startup } from '../startup';
 
 const init = async (): Promise<void> => {
+	// We don't send errors on the dev server, or if the enableSentryReporting switch is off.
+	const {
+		switches: { enableSentryReporting },
+		isDev,
+	} = window.guardian.config;
+	const dontSend = isDev || !enableSentryReporting;
```

Comment:
```suggestion
	const sentryDisabled = isDev || !enableSentryReporting;
```

What do you think of this?